src/main/java/hudson/plugins/scm/koji/client/KojiBuildDownloader.java

```diff
@@ -206,6 +208,23 @@ private File downloadRPM(File targetDir, Build build, RPM rpm) {
         return null;
     }
 
+    private String hashSum(File file) {
+        byte[] buffer = new byte[BUFFER_SIZE];
+        MessageDigest hashAlgorithm;
+        try {
+            hashAlgorithm = MessageDigest.getInstance("Md5");
+            try (InputStream inputStream = new DigestInputStream(new FileInputStream(file), hashAlgorithm)) {
+                while (inputStream.read(buffer) > 0) {
```

Comment:
I think you can ignore IDEA warning here and do simple `while(...);`, but I don't insist.